src/main/java/org/broadinstitute/dsm/model/KitRequestExternal.java

```diff
@@ -53,55 +55,39 @@ public static void updateKitRequest(String externalOrderStatus, long externalOrd
     }
 
     // update kit request with response of external shipper
-    public static void updateKitRequestResponse(String externalResponse, String dsmKitRequestId) {
-        SimpleResult results = inTransaction((conn) -> {
-            SimpleResult dbVals = new SimpleResult();
-            try (PreparedStatement stmt = conn.prepareStatement(SQL_UPDATE_KIT_REQUEST_EXTERNAL_SHIPPER_RESPONSE)) {
-                stmt.setString(1, externalResponse);
-                stmt.setString(2, dsmKitRequestId);
-
-                int result = stmt.executeUpdate();
-                if (result > 1) {
-                    throw new RuntimeException("Error updating kit request w/ dsm_kit_request_id " + dsmKitRequestId + " it was updating " + result + " rows");
-                }
+    public static void updateKitRequestResponse(@NonNull Connection conn, String externalResponse, String dsmKitRequestId) {
```

Comment:
using same connection now to update kit request and kit information of one external order